README.md

```diff
@@ -619,8 +628,8 @@ The date on which you receive the shares or cash payment is the **settlement dat
   You have been put on “autopilot” with respect to the timing of the tax event.
   This can be a really bad thing if, on the date of vesting, the shares are worth a lot and
   consequently you owe a lot of tax.
-- ☝️ **You don’t want to confuse “restricted stock units” with “restricted stock,”** an
-  entirely different thing (described soon).
+- ☝️ You don’t want to confuse *restricted stock units* with *restricted stock*, an entirely
+  different thing (described soon).
```

Comment:
One thing really nagging at me in this Guide is how frequently we say something to the effect of, "we'll discuss / describe / get to this soon." I had an editor once tell me that was one of the most obnoxious things you could do to a reader. It seems like this may be an issue of quality WRT our implementation of forward links. Current usage feels like a violation of "prioritizing utility" and "respecting the reader." I don't expect you to address in this PR, but do want to make sure this gets addressed in a future revision.

Reply:
Agree. In this case I think the right solution is internal section links (which is what I'm doing for this one) and/or links to definitions.